### ISSUE TITLE
Mac support

### ISSUE THREAD

**Issue by rpavlik** [OSVR/OSVR-Vive#5]
Most of the driver is cross-platform, however, some code in:
- https://github.com/OSVR/OSVR-Vive/blob/master/DriverLoader.cpp
- https://github.com/OSVR/OSVR-Vive/blob/master/FindDriver.cpp

is necessarily platform-specific (paths, details of loading dynamic libraries at runtime) and was only initially implemented for Windows. Valve is shipping SteamVR lighthouse drivers for OS X at this time.

Also, it has come to my attention that while SteamVR has the `driver_lighthouse.dylib` file in a `osx32` directory, it's actually a universal binary, so `getPlatformDirname()` here https://github.com/OSVR/OSVR-Vive/blob/master/FindDriver.cpp#L78 on Mac should be hard-coded to always append 32 regardless of build bitness. (Why would you append 32 if you're going to make it universal? just leave off the suffix... arg)

**Comment by zero-** [comment]
Just checking that is this suppose to be fixed by those merges?

I'm getting this when trying to run ViveDisplayExtractor:

```
libc++abi.dylib: terminating with uncaught exception of type osvr::vive::CouldNotLoadDriverModule: Could not load driver module: dlopen(/Users/zero/Library/Application Support/Steam/steamapps/common/SteamVR/drivers/lighthouse/bin/osx32/driver_lighthouse.dylib, 10): Library not loaded: @loader_path/libaitcamlib.dylib
  Referenced from: /Users/zero/Library/Application Support/Steam/steamapps/common/SteamVR/drivers/lighthouse/bin/osx32/driver_lighthouse.dylib
  Reason: image not found
```

Running OSX 10.10.5 and I've latest SteamVR installed.

**Comment by rpavlik** [comment]
Yes, it should work, but you have to set LD_LIBRARY_PATH or DYLIB_LOADER_PATH or something like that to include the lighthouse driver dir. No working mac here anymore so I can't test, but it's something like that. Documentation contribution on osvr-docs would be greatly appreciated.
